packages/dendron-plugin-views/package.json

```diff
@@ -100,7 +100,7 @@
     "build": "yarn build:dev && yarn build:styles",
     "buildCI": "echo nop",
     "build:dev": "yarn setup && npx cross-env BUILD_DEV=1 node scripts/build.js",
-    "build:prod": "yarn setup && node scripts/build.js",
+    "build:prod": "yarn setup && GENERATE_SOURCEMAP=false node scripts/build.js",
```

Comment:
When do we actually want sourcemaps?  When we're in development? 

The logic in webpack.config.js right now is: 

`sourceMap: isEnvProduction ? shouldUseSourceMap : isEnvDevelopment,`

If we just want it in development, can we should do `sourceMap: isEnvDevelopment` and get rid of this `GENERATE_SOURCEMAP` / `shouldUseSourceMap` flag?